### ISSUE TITLE
Metadata Search "Search in:" dialog to navigate federated zones

### ISSUE THREAD

**Issue by carrgilson** [DICE-UNC/irods-cloud-browser#176]
Where Federation is in use, the "Search in:" folder button brings up a dialog box with your local zone as the very first button of the navigation bar.

It would be useful to bring up the root of all zones as the very first button and then allow users to navigate within any zone rather than limiting them just the zone they logged in to.

This would be a usability improvement, as you can simply type the path to the federated zone you want to search within, but having an interface to navigate would be easier for the end user.

**Comment by michael-conway** [comment]
Yes, this is coming.  The metadata search was actually a feature we did 
an early push on for a consortium member, but will be expanded so that 
you can plug in sort of an 'app store' of different queries, either 
irods catalog queries like GenQuery, or external indexes like NoSQL, 
ElasticSearch, or a triple store.

In upcoming releases virtual collections will have a zone 'scope' option 
to run the query over multiple zones, so you could run a metadata query 
over a selected set of federated zones, tag or star across zones, etc.

We're also planning on getting rid of 'root' and instead having a left 
hand side collection for each zone.
